react/features/prejoin/components/dialogs/DialInDialog.js

```diff
@@ -63,12 +112,13 @@ function DialinDialog(props: Props) {
         passCode,
         t
     } = props;
+    const classes = useStyles();
     const flagClassName = `prejoin-dialog-flag iti-flag ${getCountryCodeFromPhone(
         number
     )}`;
 
     return (
-        <div className = 'prejoin-dialog-dialin'>
+        <div className = { clsx('prejoin-dialog-dialin', classes.root) }>
```

Comment:
Do we still need the `prejoin-dialog-dialin` class here? Is it used by tests? If not let's remove it please